node/cli/src/factory_impl.rs

```diff
@@ -162,7 +162,6 @@ impl RuntimeAdapter for FactoryState<Number> {
 	}
 
 	fn minimum_balance() -> Self::Balance {
-		// TODO get correct amount via api. See #2587.
```

Comment:
what happened to this? 😀

Reply:
(I actually wanted to check it out at some point over a weekend or sth)

Reply:
My understanding is that the `ExistentialDeposit::get()` in the line below does already fetch the value from the runtime?

There was previously a hard-coded constant, but this was changed as part of https://github.com/paritytech/substrate/pull/3239. There are some more hard-coded constants though, if you search for `#2587` you'll find more mentions :-).